Update marketplace-faq-publisher-guide.md

emuench opened the pull request:
Change "cloud partner portal" and similar to "Partner Center". This same change need to be made to linked-to topics and their source paths. Edits up to immediately after the first run of bullets.

emuench committed:
Update marketplace-faq-publisher-guide.md

Change "cloud partner portal" and similar to "Partner Center". This same change need to be made to linked-to topics and their source paths. Edits up to immediately after the first run of bullets.

qianw211 reviewed (commented):
Hi Eric, 

I think we should leave the following as is, since it's about Azure Marketplace:

The Azure Marketplace offersBoth Microsoft and partners offer technical solutions and services from Microsoft and partners built to extend Azure products and services. The solution catalog spans several categories, including but not limited to:

Thanks!
Qian

emuench committed:
Update marketplace-faq-publisher-guide.md

Restored the sentence Qian requested to keep as-is, except still suggesting removal of "but not limited to". I still think this sentence needs rewording because the word pair "partners built" initially reads like "partners are built to".
